src/hooks/useUser.ts

```diff
@@ -0,0 +1,10 @@
+import { AuthContext } from "@/context/AuthContext";
+import { useContext } from "react";
+
+export function useUser() {
```

Comment:
I _think_ you should be fine using `db.useUser` the way you have everything written. I may be wrong though